setup.cfg

```diff
@@ -24,7 +24,7 @@ max-line-length = 100
 # -p no:django - disable the pytest-django plugin for Socorro tests
 addopts = -rsxX --tb=native -p no:django
 norecursedirs = .git docs config docker __pycache__
-testpaths = socorro/unittest/
+testpaths = socorro/
```

Comment:
We now have tests in other places, so opening this up a bit allows those tests to run.